src/CMakeLists.txt

```diff
@@ -1,12 +1,11 @@
 add_subdirectory (error)
 
-
 set (ADDTESTING_PHASE OFF)
-add_subdirectory (plugins)
```

Comment:
Looks switching `add_subdirectory (plugins)` and `add_subdirectory (bindings)` broke the full build. At least building with `BUILD_FULL=ON` works on my machine, if I undo your changes to this file.